### ISSUE TITLE
alternative error Deletion failed, path = '/home/xxx/.pub-cache/_temp' 

### ISSUE THREAD

**Issue by a14n** [google/test_runner.dart#25]
I faced this following error for one of my N tests on the same code base with `0.2.14+1`. Not easy to reproduce. Perhaps a concurrent issue with the Dart Editor ? 

```
test_runner --skip-browser-tests

Checking Dart SDK binaries...
Dart SDK binaries OK.

Looking for Dart project in "./"...
Found project "rational".

Looking for test suites...
Found 3 test suites (2 Standalone VM, 1 Dartium).  
Dartium tests will be skipped!

Running all tests...
Test suite failed: rational_test.dart
Detailed results of test suite rational_test.dart:
┌─────────────────────────────────────────────────
│ 
│ Deletion failed, path = '/home/xxx/.pub-cache/_temp' (OS Error: No such file or directory, errno = 2)
└─────────────────────────────────────────────────
Test suite passed: bigint_test.dart

```

**Comment by a14n** [comment]
Note that I don't succeed to reproduce it.

**Comment by nicolasgarnier** [comment]
I've seen this a few times too... It's hard to reproduce and it seems tu be caused by `pub run`. We're going to switch from `pub run` to `dart` soon to run VM tests and this should get rid of this issue.

**Comment by nicolasgarnier** [comment]
Hey @a14n I just switched to using `dart` instead of `pub run` for running VM tests. That should not impact the Travis CI integration you've built as long as the `dart` executable is in the PATH.

We've decided to do that because there are a lot of issues with `pub run` like the fact that it runs in checked mode and the fact that `Platform.script.resolve(...)` can't work.
A nice side effect is that hopefully this error should not happen anymore.

Only thing we are loosing is auto-execution of Transformers but they are rarely used on VM code.
